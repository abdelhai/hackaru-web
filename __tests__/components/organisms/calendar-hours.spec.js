import Factory from '@/__tests__/__setups__/factory';
import CalendarHours from '@/components/organisms/calendar-hours';

describe('CalendarHours', () => {
  it('render correctly', () => {
    const wrapper = new Factory(CalendarHours, {
      provide: {
        pxPerMin: 40 / 60
      }
    }).shallow();
    expect(wrapper.element).toMatchSnapshot();
  });
});
