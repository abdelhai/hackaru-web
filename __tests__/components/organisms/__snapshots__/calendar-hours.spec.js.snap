// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalendarHours render correctly 1`] = `
<section
  class="calendar-hours"
>
  <time
    style="height: 40px;"
  >
    
    00
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    01
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    02
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    03
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    04
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    05
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    06
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    07
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    08
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    09
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    10
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    11
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    12
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    13
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    14
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    15
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    16
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    17
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    18
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    19
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    20
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    21
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    22
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
  <time
    style="height: 40px;"
  >
    
    23
    
    <span
      class="min"
    >
      :00
    </span>
  </time>
</section>
`;
